modules/porous_flow/test/tests/dirackernels/squarepulse1.i

```diff
@@ -91,6 +91,7 @@
 [Executioner]
   type = Transient
   solve_type = Newton
+  nl_abs_tol = 1e-14
```

Comment:
This is the only test that actually *does* anything.  Since the initial |R|>0.01 this nl_abs_tol is fine.

Reply:
@WilkAndy then your tests are not designed as well as they could be. You should be using `Problem/solve = false` if you don't want iterations to be taken, not rely on your initial residual to be zero. Apparently this is not a stable way of ensuring no iterations. The tolerance fudges are a band-aid here.